openshift/console
# adds support for export app in topology

**Fixes**: 
https://issues.redhat.com/browse/ODC-6213

**Solution Description**: 
adds support for export app in topology

**Screen shots / Gifs for design review**: 
![image](https://user-images.githubusercontent.com/5129024/129913000-3d941a96-6c2c-4c9f-8954-4dd6d713110a.png)

![image](https://user-images.githubusercontent.com/5129024/129912952-4087667e-f8b4-4f40-bca3-0238e2f71853.png)

![image](https://user-images.githubusercontent.com/5129024/129913082-84e4675b-65dc-49e1-bc50-b94880fe49f4.png)


**Unit test coverage report**: 


![image](https://user-images.githubusercontent.com/5129024/130182908-b8e123ff-509a-4fae-80b1-7a67bab81afc.png)

**Test setup:**
 - git clone git@github.com:cooktheryan/gitops-primer.git
 - make install
 - make run
 - make deploy
 - should be able to view `Export Application` in topology filter bar


Repo link : 
https://github.com/cooktheryan/gitops-primer
https://github.com/cooktheryan/gitops-primer#deploying


**Browser conformance**:
